modules/economy.js

```diff
@@ -1,293 +1,235 @@
-/* eslint-disable node/no-callback-literal */
 const StringPlaceholder = require('string-placeholder')
-const i18n = require('../i18n/i18n')
 
 module.exports = {
   getDailyMoney: async (client, member, guild, callback) => {
     const EgM = client.Sentry.startTransaction({
       op: 'economy.getDailyMoney',
       name: 'Economy (getDailyMoney)'
     })
-    const plusNumber = Math.round(Math.random() * (100 - 5) + 5)
+    const dailyMoney = Math.round(Math.random() * (100 - 5) + 5)
 
-    client.pool.query('SELECT * FROM `guildEconomyUserBank` WHERE guild = ? AND member = ?', [guild.id, member.id], (err, result) => {
-      if (err) {
-        client.logError(err)
-        client.log.error(err)
-      }
-      if (Object.prototype.hasOwnProperty.call(result, 0)) {
-        client.pool.query('UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?', [(parseInt(result[0].amount) + plusNumber), member.id, guild.id], (err) => {
-          if (err) {
-            client.logError(err)
-            client.log.error(err)
-          }
-        })
-      } else {
-        client.pool.query('INSERT INTO `guildEconomyUserBank` (`member`, `guild`, `amount`) VALUES (?, ?, ?)', [member.id, guild.id, plusNumber], (err) => {
-          if (err) {
-            client.logError(err)
-            client.log.error(err)
-          }
-        })
-      }
+    module.exports.getMemberInventoryAndBalance(client, member, guild, (memberInventoryAndBalance) => {
+      client.pool.query('UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?', [(parseInt(memberInventoryAndBalance.amount) + dailyMoney), member.id, guild.id], (err) => {
+        if (err) client.logError(err)
+        if (err) throw new Error('DB_ERROR')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/alexandrumuntenas/pingu/pullRequest?prid=8657799)
```suggestion
        if (err) throw new Error("DB_ERROR")
```